cloud/shared/bin/lib.sh

```diff
@@ -1,9 +1,18 @@
 #!/usr/bin/env bash
 
+set -e
+set -o pipefail
+
+export TF_VAR_FILE="setup.auto.tfvars"
+export BACKEND_VARS_FILENAME="backend_vars"
```

Comment:
can we update the config loader to get these form the env as they are stored in the setup class currently

Reply:
done, PTAL i *think* i did it correctly